frontend/src/lib/WebsocketConnection.test.tsx

```diff
@@ -521,22 +544,6 @@ describe("doInitPings", () => {
 })
 
 describe("WebsocketConnection", () => {
-  const MOCK_SOCKET_DATA = {
```

Comment:
this mock data is repeated a bunch of times through the tests. I needed to modify it to include a `sessionInfo` instance, so I pulled it into a new `createMockArgs` function, above.